pandas/core/frame.py

```diff
@@ -6546,109 +6544,133 @@ def drop_duplicates(
         ignore_index: bool = ...,
     ) -> DataFrame | None: ...
 
-    def drop_duplicates(
-        self,
-        subset: Hashable | Sequence[Hashable] | None = None,
-        *,
-        keep: DropKeep = "first",
-        inplace: bool = False,
-        ignore_index: bool = False,
-    ) -> DataFrame | None:
-        """
-        Return DataFrame with duplicate rows removed.
-
-        Considering certain columns is optional. Indexes, including time indexes
-        are ignored.
-
-        Parameters
-        ----------
-        subset : column label or sequence of labels, optional
-            Only consider certain columns for identifying duplicates, by
-            default use all of the columns.
-        keep : {'first', 'last', ``False``}, default 'first'
-            Determines which duplicates (if any) to keep.
-
-            - 'first' : Drop duplicates except for the first occurrence.
-            - 'last' : Drop duplicates except for the last occurrence.
-            - ``False`` : Drop all duplicates.
-
-        inplace : bool, default ``False``
-            Whether to modify the DataFrame rather than creating a new one.
-        ignore_index : bool, default ``False``
-            If ``True``, the resulting axis will be labeled 0, 1, …, n - 1.
 
-        Returns
-        -------
-        DataFrame or None
-            DataFrame with duplicates removed or None if ``inplace=True``.
+def drop_duplicates(
```

Comment:
You have dedented the method `drop_duplicates`.

Reply:
it seems my code decided to take a step back 😶‍🌫️
 I'll get it back in line. Thanks for catching that!